src/lib/globals.ts

```diff
@@ -79,6 +84,10 @@ if (globalThis._azleInsideCanister) {
         callbacks: {}
     };
 
+    globalThis._azleCanisterMethodIdlTypes = {};
```

Comment:
Did you initialize these in stable global?

Reply:
done